connector-runtime/connector-runtime-core/src/main/java/io/camunda/connector/runtime/core/ConnectorUtil.java

```diff
@@ -33,11 +34,27 @@ private ConnectorUtil() {}
 
   public static Optional<OutboundConnectorConfiguration> getOutboundConnectorConfiguration(
       Class<? extends OutboundConnectorFunction> cls) {
-    return Optional.ofNullable(cls.getAnnotation(OutboundConnector.class))
-        .map(
-            annotation ->
+    Map<String, String> env = System.getenv();
+    var annotation = Optional.ofNullable(cls.getAnnotation(OutboundConnector.class));
+    if (annotation.isPresent()) {
+      final var normalizedConnectorName =
+          "CONNECTOR_" + toNormalizedConnectorName(annotation.get().name()) + "_TYPE";
```

Comment:
Please extract into a static method and use it for inbound and outbound

Reply:
I didn't want to have 2 constants, that would be ugly. On the other hand, since this code is executured during startup, I can use `String.format`.

Reply:
Not sure what you mean. I meant that his code exists in both methods for inbound and outbound and can be reused.

Reply:
The code is fine its just duplicated.